app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,48 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def edit
+    load_partner
+    load_associations
+  end
+
+  def update
+    load_partner
+    load_associations
+    build_partner
+    save_partner or render 'edit'
+  end
+
+private
+  def load_partner
```

Comment:
Just semantics, but what about `assign` in favour of `load`? It seems more intention revealing.